src/components/Image.js

```diff
@@ -39,17 +39,21 @@ export const Image = ({ source, style, PlaceholderContent }) => {
         style={style}
         PlaceholderContent={PlaceholderContent}
         placeholderStyle={{ backgroundColor: colors.transparent }}
+        accessible={true}
+        accessibilityLabel={`${PlaceholderContent}`}
       />
     );
   }
-
+  // TODO Image is not accessible here, why ?
   return (
```

Comment:
we need content for PlaceholderContent

Reply:
What do you mean with PlaceholderContent?

Reply:
PlaceholderContent I thought this is a sentence that describe a bit the image in case it's not visible for some reasons ... oder ?

Reply:
i have to say, i do not know what `PlaceholderContent` is :)
but it seems to be not a correct object to pass in `accessibilityLabel`. this needs to be checked again!

Reply:
So my idea is to have an image + an image short description that is coming per server ( which our clients are providing ) with the aim of making picture accessible as well . What do you think ?

Reply:
you are right, we do not have the information about the image. this needs to come aside with the image data.
let us check if we already get some image descriptions in the whole data, that we could use here.

Reply:
there is often a `captionText` for images, we can use